django/lex_experiment/templates/lex_experiment/lex_experiment.html

```diff
@@ -24,7 +24,7 @@
           hx-indicator="#spinner"
           onsubmit="document.getElementById('submit').disabled = true;">
       {% csrf_token %}
-      <h2 class="h3" id="upload-label">{% trans "Upload scanned Notice(s) of Appeal" %}</h2>
+      <h2 class="h3" id="upload-label">{% trans "Upload Scanned Notice(s) of Appeal" %}</h2>
```

Comment:
They don't have to be scanned for this to work - If there's a chance this might confuse the users, just remove the word "scanned".